lab-sam/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "lab-03-sam",
+  "version": "1.0.0",
+  "description": "401's third lab",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
If you define the test file you can type 'node test' as another way to test your stuff.  I recommend fixing this when you get the chance.